src/app/feed/user-info-box/user-info-box.component.html

```diff
@@ -45,15 +45,15 @@ <h2>{{ apiResponseUser.name }}</h2>
 					</li>
 					<li class="list-item">
 						<span class="item-label">Website:</span>
-						<span class="item-content" *ngIf="apiResponseUser.entities.url"><a href="{{ apiResponseUser.entities.url.urls[0].url }}">{{ apiResponseUser.entities.url.urls[0].expanded_url }}</a></span>
+						<span class="item-content" *ngIf="apiResponseUser.entities"><a *ngIf="apiResponseUser.entities.url" href="{{ apiResponseUser.entities.url.urls[0].url }}">{{ apiResponseUser.entities.url.urls[0].expanded_url }}</a></span>
 					</li>
 				</ul>
 			</div>
 		</div>
 		<div class="followers gallery" *ngIf="apiResponseUserFollowers.length > 0">
 			<h3>Followers</h3>
 			<ul class="gallery-container">
-				<li class="gallery-item"  *ngFor="let item of (sortedApiResponseUserFollowers.slice(0,12)); let i = index ">
+				<li class="gallery-item"  *ngFor="let item of (sortedApiResponseUserFollowers.slice(0,3)); let i = index ">
 					<a [routerLink]="['/search']" [queryParams]="{ query : 'from:' + item.screen_name }">
 						<div class="item-image">
 							<img src="{{ item.profile_image_url }}"/>
```

Comment:
Since images are getting loaded slowly, I tried to load them with lazy-image, but it seems to give some errors and images of some profiles returns 404. Working on it.